experimenter/experimenter/nimbus_ui_new/forms.py

```diff
@@ -510,16 +556,21 @@ class Meta:
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-        self.NimbusNimbusBranchFormSet = inlineformset_factory(
+        self.NimbusBranchFormSet = inlineformset_factory(
             NimbusExperiment,
             NimbusBranch,
             form=NimbusBranchForm,
             extra=0,
         )
-        self.branches = self.NimbusNimbusBranchFormSet(
-            data=self.data or None,
-            instance=self.instance,
-        )
+
+        branches_formset_kwargs = {
+            "data": self.data or None,
+            "instance": self.instance,
+        }
+        if hasattr(self, "files") and self.files:
+            branches_formset_kwargs["files"] = self.files
```

Comment:
```suggestion
        if files := getattr(self, "files", None):
            branches_formset_kwargs["files"] = files
```

Reply:
Is there a case where the .files attr wont exist?

Reply:
Oh that's a good question.  I thought there was but it's always there:

https://github.com/django/django/blob/994950e886ea861aa862bff25a6865baa302bd32/django/forms/forms.py#L90